ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-edge1.cfg

```diff
@@ -308,8 +289,16 @@ ip security
       shared-key 7 0007054B145A1F0E0928424A0C0B4812160C09551511170B121907214A333B286214687C782720215B0B67637B7B666B3873293274733B31233B6D2A332315696A
       dpd 10 60 clear
    !
+   profile ONE-PROFILE-TO-CONTROL-THEM-ALL
```

Comment:
This test should be done on a device with HA to reflect the actual behavioral change.

Reply:
yup this is on my todo list - will move this to draft